cg/cli/workflow/mip_dna/base.py

```diff
@@ -101,15 +102,9 @@ def mip_dna(
         )
         # Commit the updates to request flowcells
         dna_api.db.commit()
-        return
-
-    # Invoke full workflow
-    context.invoke(link, case_id=case_id)
-    context.invoke(panel, case_id=case_id, dry_run=dry_run)
-    context.invoke(config_case, case_id=case_id, panel_bed=panel_bed, dry_run=dry_run)
-    context.invoke(
-        run, case_id=case_id, priority=priority, email=email, start_with=start_with, dry_run=dry_run
-    )
+        raise FlowcellsNeededError(
+            "Analysis cannot be started: all flowcells need to be on disk to run the analysis"
+        )
```

Comment:
Should we add a log message that all flowccells are present on disk?

```suggestion
        )
    LOG.info("All flowcells present on disk")
```